.buildkite/pipeline.deploy.yml

```diff
@@ -12,6 +12,13 @@ steps:
         login: true
         account-ids: ${ECR_ACCOUNT_ID}
 
+  - name: ":ecr: ECR Vulnerabilities Scan"
+    command: "true"
+    soft_fail: true
+    plugins:
```

Comment:
does this need a `depends_on` to ensure it runs after the image is pushed?

Reply:
Yes. Updated here https://github.com/buildkite/docs/pull/2519/commits/6ce10328bfc0350cd5cf49d91687ecf85aedeafd.